src/main/xar-resources/data/urlrewrite/urlrewrite.xml

```diff
@@ -505,35 +565,32 @@ declare variable $exist:root external;</programlisting>
-        <literal>web.xml</literal>, we would need to make sure that this path is ignored in our main
-        <literal>controller.xq</literal>. However, if the mapping is done via
-        <literal>controller-config.xml</literal>, XQueryURLRewrite will already have handled the
-      path, which then won't need to be accounted for in our controller.</para>
+    <para>The <tag>forward</tag> elements specify path mappings for common servlets, similar to the
+      servlet mapping in <literal>$EXIST_HOME/etc/webapp/WEB-INF/web.xml</literal>. The advantage to
+      specifying them in <literal>controller-config.xml</literal> is that the XQueryURLRewrite
+      servlet (which implements the URL Rewriting framework) becomes a single point of entry for the
+      entire web application, and no additional handling of the servlet paths is necessary in the
+      controller script.</para>
+    <para>For example, if we had registered a servlet mapping for <literal>/rest</literal> in
+        <literal>web.xml</literal>, we would jave needed to make sure that this path is ignored in
+      our main <literal>controller.xq</literal> scripts. However, since the mapping is done via
+        <literal>controller-config.xml</literal>, XQueryURLRewrite handles the path, which then
+      doesn't need to be accounted for in our controller.</para>
     <para>The <tag>root</tag> elements define the roots of a directory or database collection
       hierarchy, mapped to a certain base path. For example, the default
         <literal>controller-config.xml</literal> uses two roots:</para>
     <programlisting language="xml" xlink:href="listings/listing-7.txt"/>
-    <para>This means that paths starting with <literal>/tools</literal> will be mapped to the
-      collection hierarchy below <literal>/db/www</literal>. Everything else is handled by the catch
-      all pattern pointing to the root directory of the webapp (by default corresponding to
-        <literal>$EXIST_HOME/etc/webapp</literal>). For example, the URL</para>
-    <programlisting>http://localhost:8080/exist/tools/admin/admin.xq</programlisting>
-    <para>will be handled by the controller stored in database collection
-        <literal>/db/www/admin/</literal> (if there is one) or will directly resolve to
-        <literal>/db/www/admin/admin.xq</literal>. In this case, all relative or absolute URLs
-      within the controller will be resolved against the database, not the file system. However,
-      there's a possibility to escape this path interpretation as described <link
-        xlink:href="#EXTERNAL_RESOURCES">below</link>.</para>
+    <para>This means that paths starting with <literal>/apps</literal> will be mapped to the
+      collection hierarchy below <literal>/db/apps</literal>. All relative or absolute URLs within
+      the URL space managed by the <literal>controller.xq</literal> script will be resolved against
```

Comment:
s.a. "resolve against ..."

Reply:
Sorry, I don't follow?

Reply:
```suggestion
      the URL space managed by the <literal>controller.xq</literal> script resolve against
```

Reply:
avoid overusing passive voice, generally good advice not just for technical writing

Reply:
Ah, it was a suggestion of active vs. passive! I agree that this is sound general advice, but not needed in this case. The current formulation is clear, and changing it from passive to active voice would require introducing a subject. Simply saying "the path will resolve" is no clearer; we'd need to say "*eXist* will resolve the path" (or *the system* or *the servlet*). I avoid contorting a sentence just to achieve an active voice, if the thought is already clear in its passive form given the context (an article about how *eXist* works). But let me know if you think there is some ambiguity or room for misunderstanding.